README.md

```diff
@@ -2,74 +2,145 @@
+- Lua-compiled Nelua implementations
+
+Master branch benchmark runs are recorded through
+[Github Actions](https://github.com/edubart/nelua-lang/actions).
+
+### Exemplary measurement reports 
+
+- LuaJIT 2.1.0-beta3
+- GCC 9.3.0 `-O3 -fno-plt -march=native -flto`
+- Lua 5.3.5
+- CPU Intel Core i7-3770K CPU @ 3.50GHz
+- Arch Linux
+
+|    benchmark |  lua 5.3 | luajit 2.1 |    nelua |        c |
+|--------------|----------|------------|----------|----------|
+|    ackermann | 2448.2 ms | 145.2 ms  |  47.8 ms |  47.4 ms |
+|    fibonacci  | 2612.4 ms | 951.7 ms  | 279.9 ms | 280.9 ms |
+|       mandel | 2549.6 ms |  97.0 ms  |  88.5 ms |  88.2 ms |
+|        sieve | 1240.4 ms | 265.3 ms  |  88.7 ms |  60.1 ms |
+|     heapsort | 2602.1 ms | 274.0 ms  | 170.9 ms | 127.5 ms |
+
+*NOTE*: Nelua could match C speed if all benchmarks were coded using optimized structures,
+but to make fair comparisons with Lua/LuaJIT they were coded in Lua style
+(using sequence tables and a garbage collector).
 
 ## About
 
 Nelua is a [systems programming language](https://en.wikipedia.org/wiki/System_programming_language)
-for performance sensitive applications where
-[Lua](https://en.wikipedia.org/wiki/Lua_(programming_language))
-would not be efficient, like operational systems, real-time applications and game engines.
-It has syntax and semantics similar to Lua,
-but is designed to be able to work free from a Lua interpreter,
-instead it takes advantage of
-[ahead of time compilation](https://en.wikipedia.org/wiki/Ahead-of-time_compilation).
-When coding using Nelua idioms such as type notations, records, arrays,
-manual memory management, pointers the performance should be efficient as C.
-But when using Lua idioms such as tables, metatables and untyped variables the compiler
-uses a runtime library to provide the dynamic functionality.
-
-Nelua can do compile-time [meta programming](https://en.wikipedia.org/wiki/Metaprogramming)
-because it has a Lua preprocessor
-capable to cooperate with the compiler as it compiles,
-this is only possible because the compiler is fully made in Lua
-and is fully accessible or modifiable by the preprocessor on the fly.
-Therefore it's possible to implement higher constructs such as classes,
+for performance-sensitive applications where [Lua](https://en.wikipedia.org/wiki/Lua_(programming_language))
+would not be efficient, such as operating systems, real-time applications and
+game engines. While it has syntax and semantics similar to Lua, it primarily
+focuses on generating efficient C code and provide support for
+highly-optimizable low-level programming. Using Nelua idioms such as records,
+arrays, manual memory management and pointers should result in performance as
+efficient as pure C; on the other hand, when using Lua idioms such as tables,
+metatables and untyped variables, the compiler will bake a runtime library for
+this sort of dynamic functionality into the program, which could incur some
+noticeable runtime cost.
```

Comment:
I like the changes heres, except the `noticeable runtime cost` could be replaced with `some runtime overhead`, not all dynamic functionally incur noticeable runtime cost, some are very minor.

Reply:
Changed to "some runtime overhead"